terraform-cli/terraform-cli-task.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: terraform-cli
+spec:
+  inputs:
+    resources:
+      - name: source
+        type: git
+    params:
+      - name: ARGS
+        description: The terraform cli commands to tun
+        type: array
+        default:
+          - "--help"
```

Comment:
Why would be the default as --help ? Do you want to make sure it fails if the user don't specify the ARGS?

Reply:
I'm using this as default just like the OC client task and the tower CLI task that i previously submitted, so the command doesn't run indefinitely

Reply:
ah i didnt' know about this pattern 🤔  cc @vdemeester